pex/testing.py

```diff
@@ -58,8 +58,12 @@
 NOT_CPYTHON27 = IS_PYPY or PY_VER != (2, 7)
 IS_LINUX = platform.system() == "Linux"
 IS_MAC = platform.system() == "Darwin"
-IS_NOT_LINUX = not IS_LINUX
-NOT_CPYTHON27_OR_OSX = NOT_CPYTHON27 or IS_NOT_LINUX
+machine = platform.machine()
+IS_LINUX_X86_64 = IS_LINUX and machine == "x86_64"
```

Comment:
When I had to skip a handful of tests that implicitly assume they're running on Linux x86_64, I did so by subtracting ("run unless on aarch64") instead of "only run on x86_64" so that if we add more platforms in the future (notably, Windows) these tests will fail on those platforms and at least force us to consider what to do with them.